test/resources/statefulset.go

```diff
@@ -148,10 +159,8 @@ func (t *TestStatefulset) CreateWithoutWaiting() {
 	}
 }
 
-func (t *TestStatefulset) WaitForPodReady() {
-
-	err := waitForStatefulSetComplete(t.Client, t.Namespace, t.Statefulset)
-	framework.ExpectNoError(err)
+func (t *TestStatefulset) WaitForPodReady() error {
```

Comment:
It would be good to move this function near to its internal implementation for easier discovery.